tests/testapp/tests/test_connection.py

```diff
@@ -103,6 +114,24 @@ def test_csr(self):
             self.network_connection.certificate_signing_request(self.root_cert, '', '')
         self.assertTrue(Certificate.objects.filter(id=json.loads(cert_serialized)['id']).exists())
 
+    @mock_patch_decorator
+    def test_push_signed_client_certificate_chain(self):
+        # sanity test to make sure things are working as expected
+        NetworkSyncConnection._request.return_value.json.return_value = {'id': uuid.uuid4().hex, 'public_key': Key().get_public_key_string()}
+        cert = self.network_connection.push_signed_client_certificate_chain(self.root_cert,
+                                                                            self.subset_scope_def.id,
+                                                                            {"mainpartition": self.root_cert.id, "subpartition": "abracadabra"})
```

Comment:
Wait, so what is this test testing, exactly? The API endpoints are mocked, so it's just testing that the `push_signed_client_certificate_chain` method is able to complete?

Reply:
yes was thinking this as a smoke test.